### PR TITLE
python 3.11

### PR DESCRIPTION
**Pull request by fcollman**

### CODE REVIEW

**Comment by fcollman** [comment]
```
module 'numpy' has no attribute 'int'. `np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information. The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations```

**Comment by fcollman** [comment]
remeshing error..

```2023-09-20 05:01:51.022 PDT
Traceback (most recent call last): File "/app/venv/lib/python3.11/site-packages/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py", line 126, in _wrap_callback_errors callback(message) File "/app/venv/lib/python3.11/site-packages/messagingclient/client.py", line 36, in callback_wrapper callback(payload) File "/app/workers/mesh_worker.py", line 53, in callback meshgen.remeshing( File "/app/pychunkedgraph/meshing/meshgen.py", line 891, in remeshing chunk_stitch_remeshing_task( File "/app/pychunkedgraph/meshing/meshgen.py", line 1111, in chunk_stitch_remeshing_task multi_child_nodes, _ = get_multi_child_nodes(cg, chunk_id, node_id_subset, False)
2023-09-20 05:01:51.023 PDT
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-20 05:01:51.023 PDT
File "/app/pychunkedgraph/meshing/meshgen.py", line 1059, in get_multi_child_nodes
2023-09-20 05:01:51.023 PDT
multi_child_descendants = meshgen_utils.get_downstream_multi_child_nodes(
2023-09-20 05:01:51.023 PDT
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-20 05:01:51.023 PDT
File "/app/pychunkedgraph/meshing/meshgen_utils.py", line 137, in get_downstream_multi_child_nodes
2023-09-20 05:01:51.023 PDT
return recursive_helper(node_ids)
2023-09-20 05:01:51.023 PDT
^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-20 05:01:51.023 PDT
File "/app/pychunkedgraph/meshing/meshgen_utils.py", line 126, in recursive_helper
2023-09-20 05:01:51.023 PDT
children_array = np.array(list(node_to_children_dict.values()))
2023-09-20 05:01:51.023 PDT
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (22,) + inhomogeneous part.
```

**Comment by fcollman** [comment]
this is working on ltv and deployed and working on FANC/BANC now. @sdorkenw @akhileshh do you think we should merge it now?

File: Dockerfile [hunk 1]
```diff
@@ -63,6 +61,8 @@ COPY --from=bigtable-emulator-build /go/bin/emulator /app/venv/bin/cbtemulator
 COPY override/gcloud /app/venv/bin/gcloud
 COPY override/timeout.conf /etc/nginx/conf.d/timeout.conf
 COPY override/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+# Hack to get zstandard from PyPI - remove if conda-forge linked lib issue is resolved
```
Comment (ranlu) [inline Dockerfile:64]: The direct cause of the conda-forge issue is https://github.com/conda-forge/zstandard-feedstock/pull/45, it choose to link the system zstd to solve the license issue mentioned in https://github.com/conda-forge/zstandard-feedstock/issues/43

Comment (ranlu) [reply Dockerfile:64]: The `multi_decompress_to_buffer` function uses internal pool api, which is only enabled when the package is compiled with bundled zstd.

**Pull request merged**